gatling-core/src/main/scala/io/gatling/core/session/package.scala

```diff
@@ -72,26 +78,22 @@ package object session {
     resolveRec(_, iterable.iterator, Nil)
   }
 
-  def seq2SeqExpression(seq: Seq[(String, Any)]): Expression[Seq[(String, Any)]] = {
-    val elValues: Seq[(String, Expression[Any])] = seq.map { case (key, value) =>
-      val elValue = value match {
-        case s: String => s.el[Any]
-        case v         => v.expressionSuccess
+  def seq2SeqExpression(seq: Seq[(Any, Any)]): Expression[Seq[(Any, Any)]] = {
```

Comment:
See below, I think it should be seq2SeqExpression(seq: Seq[(String, Any)]): Expression[Seq[(String, Any)]]